src/modules/m_hideoper.cpp

```diff
@@ -28,25 +28,38 @@
 class HideOper : public SimpleUserModeHandler
 {
  public:
+	size_t opercount;
+
 	HideOper(Module* Creator) : SimpleUserModeHandler(Creator, "hideoper", 'H')
+		, opercount(0)
 	{
 		oper = true;
 	}
+
+	ModeAction OnModeChange(User* source, User* dest, Channel* channel, std::string& parameter, bool adding)
+	{
+		if (dest->IsModeSet(this->GetModeChar()) != adding)
```

Comment:
Instead of reimplementing `SimpleUserModeHandler::OnModeChange()`, just call it and if it returns `MODEACTION_ALLOW` do the increment/decrement. This makes it so if SimpleUserModeHandler changes then this won't break.